packages/manager/modules/log-to-customer/src/list/controller.js

```diff
@@ -30,18 +38,26 @@ export default class LogToCustomerListCtrl {
 
         this.selectedAccount =
           this.accountList.length > 0 ? this.accountList[0] : null;
+        this.buildLDPHomePageLink();
         this.getStreamData();
       })
       .catch((error) => {
-        this.errorMessage = error?.data?.message;
+        this.Alerter.error(error?.data?.message);
```

Comment:
You might provide a generic message error + the  detail of the error to avoid having empty error alert.

```
error: "an error occurs {{ message }} "
```
As it's used several times, consider to create a function
```
function alertError(message = '') {
 this.Alterter.error(this.$translate.instant('error.key', {message}));
}
```